### DISCUSSION TITLE
Auth?

### DISCUSSION THREAD

**Discussion by ghost** [webdev-20/shorti-url-shortener#108] [General]
Hi @cherylli . Just want some clarification on the auth situation in this project. I see google auth. as a ticket. And I see user model has email & password in `server`. 

I think both can be done. I'd like to try to do one of them if no one else is working on it. 

What do you think? Do you have any ideas what the auth should be like or what you'd like to see done? Should this be worked on now or maybe later after some other higher priority tasks are addressed? Etc etc etc

**Comment by cherylli** [comment]
Hi, thanks for asking. I'm currently working on frontend auth (slowly) #95 , which I think is 80% done 
@Ash1eyC0des suggested to add google auth to the backend (#59), but i'm not sure if she's still working on it. If she is not, you're welcome to implement it in the backend, and the frontend after I'm done with the basic username/password auth in the frontend 

There are other things higher priority than auth, if you look at the [Issues board](https://github.com/webdev-20/shorti-url-shortener/issues), I would like to close most of the tickets before working on new features
